docker/jenkins/sign-release.sh

```diff
@@ -109,17 +109,38 @@ elif [ "$EXT" == "rpm" ]; then
     fi
     touch "$RPM_MACROS"
     echo "%_signature gpg"  >> $RPM_MACROS
+
+    # define key name and directory where we installed temporaries
     echo "%_gpg_name $KEY_ID" >> $RPM_MACROS
     echo "%_gpg_path $TMP_KEYRING_DIR" >> $RPM_MACROS
-    cat $RPM_MACROS
 
-    # perform the actual signature
-    /usr/bin/expect << EOD
+    if [ -f /etc/fedora-release ]; then
```

Comment:
I'm not well-read on current standards, but I've been trying to consolidate on using values from `/etc/os-release` whenever I can for OS detection; it seems that all recent releases are providing this file.

Reply:
Unfortunately this script runs on release which do not!